pages/api/[[...slug]].js

```diff
@@ -42,22 +42,27 @@ const resolvers = {
   Mutation: {
     ...carsMutation,
   },
-
 };
 
 const server = new ApolloServer({ typeDefs, resolvers });
 
 const handler = server.createHandler({
-  path: "/api/graphql",
+  path: '/api/graphql',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/bandvov/first-nextjs/pullRequest?prid=7138117)
```suggestion
  path: "/api/graphql",
```